.github/workflows/lint.yml

```diff
@@ -10,14 +10,17 @@ jobs:
   golangci:
     strategy:
       matrix:
-        go-version: [1.16.x, 1.17.x]
+        go-version: [1.17.x]
         os: [ubuntu-latest]
     name: lint
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
 
     steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ matrix.go-version }}
```

Comment:
Good take but I think we should mention why is this needed.

Reply:
Everything is wrote in the release notes of new golangci-lint-action, so I think we don't need to copy-paste the info from upstream dep